controllers/resourceController.js

```diff
@@ -0,0 +1,57 @@
+const asyncHandler = require('express-async-handler');
+const Resource = require('../models/resourceSchema') //Schema
+
+
+
+//@desc Get all resources
+//@route GET /api/resources/:keyword
+//@access Public
+const getResources = asyncHandler(async (req, res) => {
+    const keyword = req.params.keyword.toLowerCase();
+    const resources = await Resource.find({
+        keywords: keyword
+    })
+
+    //Throw error if resource doesn't exist.
+    if (!resources) {
+        res.status(400).json({
```

Comment:
```suggestion
        res.status(404).json({
```
I think a 404 resource not found status code would be better here rather than a 400 bad request 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404